src/docs/development/data-and-backend/json.md

```diff
@@ -278,18 +280,18 @@ from the previous samples.
 
 **user.dart**
 
-{% prettify dart %}
+<?code-excerpt "lib/serializable/user.dart"?>
+```dart
 import 'package:json_annotation/json_annotation.dart';
 
 /// This allows the `User` class to access private members in
 /// the generated file. The value for this is *.g.dart, where
 /// the star denotes the source file name.
-part '[[highlight]]user[[/highlight]].g.dart';
```

Comment:
I am not sure how to add the `[[highlight]]` code in here, using a "replace" regex in the code-excerpt seems very complicated. Alternatively I could revert this change and leave this code sample without a code-excerpt. I am open to ideas!

Reply:
Yes, the highlight directive is only supported in `prettify` sections. In this case, just remove the highlight. I don't want you jumping hoops with regex. Just please make sure that the introductory text to this section doesn't refer to the "highlighted" code, since it won't be.

Reply:
Got it! I will double check that

Reply:
I didn't see any references to highlighted code, so it should be good to go :)